download.dd

```diff
@@ -190,6 +190,8 @@ sudo apt-get update && sudo apt-get install dmd-compiler dub)
 
 $(DOWNLOAD_OTHER $(OPENSUSE), OpenSUSE Tumbleweed, $(CONSOLE sudo zypper install dmd))
 
+$(DOWNLOAD_OTHER $(SNAP), $(LINK2 https://snapcraft.io/dmd, Snap), $(CONSOLE sudo snap install --classic dmd))
```

Comment:
Given that you have explicit instructions for the APT repo above for separate dmd and dub install, why not either:

* a separate line for DMD snap and DUB snap, or

* `sudo snap install --classic dmd && sudo snap install --classic dub`

.... ?